### DISCUSSION TITLE
ScriptReference/Animator

### DISCUSSION THREAD

**Discussion by docscomments** [Developer-Notes-Extension/Unity-Notes#368] [Notes]
# ScriptReference/Animator

https://docs.unity3d.com/ScriptReference/Animator.html

**Comment by docscomments** [comment]
**Comment by `@Juskelis`:**  

Remember that the animation system is updated after Update and before LateUpdate. See [here](https://docs.unity3d.com/Manual/ExecutionOrder.html) for more info.

**Comment by keenanwoodall** [comment]
`MonoBehaviour.OnDidApplyAnimationProperties` is an undocumented event function you can implement to receive callbacks after animation properties are applied. 
It often makes sense to use this function in place of `OnValidate` or `LateUpdate` when responding to changed properties.
